src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileServiceImpl.java

```diff
@@ -0,0 +1,172 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.kfs.sys.batch.BatchInputFileType;
+import org.kuali.kfs.sys.batch.service.BatchInputFileService;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileValidationService;
+
+public class ConcurRequestExtractFileServiceImpl implements ConcurRequestExtractFileService {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractFileServiceImpl.class);
+    protected String acceptedDirectoryName;
+    protected String rejectedDirectoryName;
+    protected BatchInputFileType batchInputFileType;
+    protected BatchInputFileService batchInputFileService;
+    protected ConcurRequestExtractFileValidationService concurRequestExtractFileValidationService;
+
+    public List<String> getUnprocessedRequestExtractFiles() {
+        List<String> fileNamesToLoad = getBatchInputFileService().listInputFileNamesWithDoneFile(getBatchInputFileType());
+        return fileNamesToLoad;
+    }
+
+    public boolean processFile(String requestExtractFileName) {
+        boolean processingSuccess = true;
+        Object parsedFile;
```

Comment:
Since this is not used outside the try I think it would make sense to move this inside the try where the parsedFile is first used